prow/cmd/tide/main.go

```diff
@@ -78,6 +82,9 @@ func (o *options) Validate() error {
 			return err
 		}
 	}
+	if o.githubOnly && o.gerritOnly {
+		return errors.New("--github-only and --gerrit-only are mutual exclusive")
```

Comment:
```suggestion
		return errors.New("--github-only and --gerrit-only are mutually exclusive")
```

Reply:
Nit: Maybe also check that at least one of these flags is turned on? (Or default one of them to true)?

Reply:
same as above, they don't need to be provided most of the times